Makefile

```diff
@@ -5,6 +5,9 @@ FH_SYSTEM_DUMP_TOOL_VERSION := $(shell git describe --tags --abbrev=14)
 endif
 LDFLAGS := -X main.Version=$(FH_SYSTEM_DUMP_TOOL_VERSION)
 
+IMPORT_PATH := github.com/feedhenry/fh-system-dump-tool
+GO_DOCKER_IMAGE := golang:1.6
```

Comment:
This is [not available anymore](https://hub.docker.com/_/golang/). Need to test, but it should work with any newer version of Go as well.